docs/examples/pingone/role-assignment.md

```diff
@@ -11,42 +16,33 @@ The following shows an example of environment creation using the PingOne Terrafo
 !!! note "Variable Mapping"
     The example uses the `license_id` variable that can be mapped directly, or can be found from the license name from the `pingone_licenses`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/data-sources/licenses" target="_blank">:octicons-link-external-16:</a> data source.
 
-First, we will need the ID of the "My Administrators" population, which we can look up from the `pingone_population`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/data-sources/population" target="_blank">:octicons-link-external-16:</a> data source.
+First, we will create the group in PingOne that we will assign our administrator users to.  This uses the `pingone_group`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/resources/group" target="_blank">:octicons-link-external-16:</a> resource.
```

Comment:
... in PingOne **to which we will assign our administrator users.  This example uses** the